tests/unit/executor_test_base.py

```diff
@@ -188,6 +193,30 @@ def set_data(self, table, data):
         con.execute('DROP TABLE IF EXISTS {}'.format(table))
         con.execute('CREATE TABLE {} AS SELECT * FROM data'.format(table))
 
+    def wait_predictor(self, project, name, timeout=100):
```

Comment:
Maybe worth opening an issue so that we can refactor other tests to use this instead.